src/publishCommand/publishManager.ts

```diff
@@ -122,11 +123,13 @@ export class PublishManager {
         cacheType: cacheType,
         name: layerName,
         tilesPath: row.tilesPath,
-        format: row.format as TileFormat,
+        format: row.format as TileOutputFormat,
       });
     } catch (exception) {
       this.logger.error(row, 'Failed to handle row for data:');
       throw exception;
+    } finally {
+      this.logger.info(row, 'Finished to handle row');
```

Comment:
maybe add some debug log that presents the object data that was sent? sometimes it helps to understand.
not sure - asking and consulting on the flight :)

Reply:
https://mapcolonies.atlassian.net/browse/MAPCO-2803 added